spec/query_spec.cr

```diff
@@ -30,6 +30,21 @@ describe LuckyRecord::Query do
     end
   end
 
+  describe "#first?" do
+    it "gets the first row from the database" do
+      UserBox.new.name("First").save
+      UserBox.new.name("Last").save
+
+      user = UserQuery.new.first?
+      user.should_not be_nil
+      user && user.name.should eq "First"
```

Comment:
You could also remove `user &&` and do `user.not_nil!.name.should eq "First"`. I'm fine with either, but wanted to let you know about the other option in case you prefer it

Reply:
oh cool I didn't realize I could chain `not_nil!`